modules/gapi/include/opencv2/gapi/gasync_context.hpp

```diff
@@ -0,0 +1,37 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2019 Intel Corporation
+
+#ifndef OPENCV_GAPI_GASYNC_CONTEXT_HPP
+#define OPENCV_GAPI_GASYNC_CONTEXT_HPP
+
+#if !defined(GAPI_STANDALONE)
+#  include <opencv2/core/cvdef.h>
+#else   // Without OpenCV
+#  include <opencv2/gapi/own/cvdefs.hpp>
+#endif // !defined(GAPI_STANDALONE)
```

Comment:
I thought our model is just "always include own/cvdefs.hpp`" and there we have that `#ifdef` check. Isn't it?

Reply:
`<opencv2/gapi/own/cvdefs.hpp>` has no `#include <opencv2/core/cvdef.h>` and never had IIRC.

Reply:
well, probably it is worth adding then :) Not in this PR, for sure.